src/main/java/com/actiontech/dble/services/manager/response/ShowHelp.java

```diff
@@ -239,7 +237,9 @@ private static RowDataPacket getRow(String stmt, String desc, String charset) {
 
         HELPS.put("show @@statistic", "Turn off statistic information");
         HELPS.put("enable @@statistic", "Turn on statistic sql");
-        HELPS.put("disable @@statistic", "Turn off statistic sql");
+        HELPS.put("enable @@enableStatisticAnalysis", "Turn on statistic analysis sql('show @@sql.sum.user/table' or 'show @@sql.condition')");
+        HELPS.put("disable @@statistic", "Turn off statistic analysis sql('show @@sql.sum.user/table' or 'show @@sql.condition')");
```

Comment:
this is disable @@enableStatisticAnalysis

Reply:
get